tests/specifications/general_test.py

```diff
@@ -390,19 +391,52 @@ def test_check_058():
   assert bad_name3 in message
   assert good_name1 not in message
 
+  # Upgrade to post format 3.0 and roundtrip data to update TTF object.
+  test_font = TTFont(test_font_path)
+  test_font["post"].formatType = 3.0
+  test_file = io.BytesIO()
+  test_font.save(test_file)
+  test_font = TTFont(test_file)
+  status, message = list(check(test_font))[-1]
+  assert status == PASS
+  assert "format 3.0" in message
```

Comment:
then check for SKIP here

And in this case, checking for `"format 3.0" in message` would not be necessary